<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QDialog" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Угадай Город</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <widget class="QWidget" name="Game">
    <widget class="QLabel" name="TimeLabel">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>30</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 16pt &quot;Sitka Small&quot;;
color: rgb(255, 176, 49);
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="TextLbl">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>30</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 16pt &quot;Sitka Small&quot;;</string>
     </property>
     <property name="text">
      <string>Осталось:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Picture">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>600</width>
       <height>400</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic 22pt &quot;Sitka Small&quot;;
background-color: rgb(195, 195, 195);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>                    Приготовься...</string>
     </property>
    </widget>
    <widget class="QLabel" name="TextLbl_2">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>100</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 16pt &quot;Sitka Small&quot;;</string>
     </property>
     <property name="text">
      <string>Счёт:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ScoreLabel">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>100</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 16pt &quot;Sitka Small&quot;;
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PauseButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Пауза</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hint_5050">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>630</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>50/50</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hint_gym">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>630</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Помощь зала</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hint_friend">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>630</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Звонок другу</string>
     </property>
    </widget>
    <widget class="QPushButton" name="option_1">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>470</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>1 вариант</string>
     </property>
    </widget>
    <widget class="QPushButton" name="option_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>520</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>3 вариант</string>
     </property>
    </widget>
    <widget class="QPushButton" name="option_2">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>470</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>2 вариант</string>
     </property>
    </widget>
    <widget class="QPushButton" name="option_4">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>520</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>4 вариант</string>
     </property>
    </widget>
    <widget class="QLabel" name="o1_percent">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>470</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="o3_percent">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>520</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="o2_percent">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>470</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="o4_percent">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>520</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Pause">
    <widget class="QPushButton" name="Continue">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>130</y>
       <width>451</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backToMenu">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>180</y>
       <width>451</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Выход в главное меню</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Exit">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>630</y>
       <width>451</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
    <widget class="QLabel" name="Logo">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>40</y>
       <width>391</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/util_img/util_img/Logo_pink_grey.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
